src/wombats/handlers/game.clj

```diff
@@ -14,9 +13,11 @@
          :max-players 10
          :type :round
          :round-intermission (* 1000 60 20)
+         :round-length (* 1000 60)
```

Comment:
This is just an example to be used in the swagger docs for now, but can be updated as we see fit. `max-players` should be less than 8 though.. Feel free to update these if you're finding yourself constantly changing the swagger docs values, however as soon as we add the admin panel this will probably be infrequently used.